config-linux.md

```diff
@@ -99,7 +99,7 @@ The runtime may supply them however it likes (with [mknod][mknod.2], by bind mou
 
 The following parameters can be specified:
 
-* **`type`** *(string, required)* - type of device: `c`, `b`, `u` or `p`.
+* **`type`** *(string, required)* - type of device: "c", "b", "u" or "p".
   More info in [mknod(1)][mknod.1].
 * **`path`** *(string, required)* - full path to device inside container.
 * **`major, minor`** *(int64, required unless **`type`** is `p`)* - [major, minor numbers][devices] for the device.
```

Comment:
On Thu, Mar 10, 2016 at 06:36:58PM -0800, Qiang Huang wrote:
> > @@ -99,7 +99,7 @@ The runtime may supply them however it likes (with [mknod][mknod.2], by bind mou
> >  
> >  The following parameters can be specified:
> >  
> > -* **`type`** *(string, required)* - type of device: `c`, `b`, `u` or `p`.
> > +* **`type`** *(string, required)* - type of device: "c", "b", "u" or "p".
> >    More info in [mknod(1)][mknod.1].
> >  * **`path`** *(string, required)* - full path to device inside container.
> >  * **`major, minor`** *(int64, required unless **`type`** is `p`)* - [major, minor numbers][devices] for the device.
>
> You might want to change `p` here and below in `Device whitelist`
> section as well. Anyway I don't see this necessary, because I don't
> think there is inkling for a specific type with backticks since it's
> just consistent usage for markdown. See discussion in
> https://github.com/opencontainers/specs/pull/323#issuecomment-187568649

I think the style we choose is less important thant that we pick
something and document it in [1] for consistency.  Having to think
about this whenever someone inserts a string literal in new docs is
wasted time ;).

[1]: https://github.com/opencontainers/specs/blob/master/style.md